tests/e2e/test_rosacli_ingress.go

```diff
@@ -239,8 +239,12 @@ var _ = Describe("Edit default ingress",
 				By("Check that the actual ingress config matches what was specified at install")
 				ingress, _ := defaultIngress(*ingressList)
 				Expect(ingress).NotTo(BeNil())
-				Expect(ingress.ExcludeNamespace).To(Equal(ingressConfig.DefaultIngressExcludedNamespaces))
-				Expect(ingress.RouteSelectors).To(Equal(ingressConfig.DefaultIngressRouteSelector))
+				DefaultIngressInArrayFormList := strings.Split(ingressConfig.DefaultIngressExcludedNamespaces, ",")
```

Comment:
please make sure to follow golang coding convention
- `DefaultIngressInArrayFormList` -> `defaultIngressInArrayFormList`
- `DefaultIngressInArrayForm` -> `defaultIngressInArrayForm`
- `DefaultIngressRouteSelectorList` -> `defaultIngressRouteSelectorList`
- `DefaultIngressRouteSelector` -> `defaultIngressRouteSelector`

Reply:
Thanks Tristan, i have updated the variable names accordingly.